MANIFEST.in

```diff
@@ -1,5 +1,4 @@
+graft fideslog/sdk/python
```

Comment:
I double checked on my machine and this does work as expected. The `tar.gz` that gets uploaded pypi is missing the `api` files. Very nice!